app_controller/pixi_frontend/assets/css/style.css

```diff
@@ -0,0 +1,7 @@
+#joystick_area{
+    height: 100%;
+    width: 100%;
+    position: relative;
+}
+
+<style>* {padding: 0; margin: 0}</style>
```

Comment:
What is this?

Reply:
A hold-over from the pixi tutorial. My b